packages/fxa-payments-server/package.json

```diff
@@ -76,6 +76,7 @@
     "prettier": "^1.18.2",
     "redux-devtools-extension": "^2.13.8",
     "sass-lint": "^1.13.1",
+    "supertest": "4.0.2",
```

Comment:
Oh, actually, meant to say that this kinda picks the testing tool for server routes on this module, and I'm the last person who should be picking that. I can remove all the test related changes from this PR if something else would be more appropriate.

Reply:
Yeah, we should probably rework the testing here... we already have progress on using jest for this stuff (e.g. [csp tests](https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/server/lib/csp.test.js))

Reply:
Oh, but maybe this works within jest... haven't had a chance to actually pull this down and try it yet